packages/loader/driver-definitions/src/storage.ts

```diff
@@ -59,8 +71,16 @@ export interface IDeltaStorageService {
 export interface IDocumentDeltaStorageService {
     /**
      * Retrieves all the delta operations within the exclusive sequence number range
-     */
-    get(from: number, to: number): Promise<IDeltasFetchResult>;
+     * @param from - first op to retrieve (inclusive)
+     * @param to - first op not to retrieve (exclusive end)
+     * @param abortSignal - signal that aborts operation
+     * @param cachedOnly - return only cached ops, i.e. ops available locally on client.
+     */
+     readMessages(from: number,
+        to: number | undefined,
+        abortSignal?: AbortSignal,
+        cachedOnly?: boolean,
```

Comment:
cachedOnly seems a bit odd here. Do all driver support caching? i wonder if the caching approach we are taking isn't generic enough. should the loader take a cache that the delta manger (or some shared layer) uses over the driver, then we don't need to expose this here, as the cache would be above this. for snapshots this didn't make as much sense due to the format differences, but for ops it seems easily generalizable.

Reply:
Caching can come from different angles. ops cached in SPO snapshot is one such form (that does not fit loader being involved).
While I agree that from API perspective it looks a bit out of place, driver implement this interface just fine by returning empty result when asking for only cached ops

Reply:
is it used anywhere right now? i expected maybe delta manager if the state is paused or something. what scenarios is it for?

Reply:
It's coming in couple next PRs - see https://github.com/microsoft/FluidFramework/pull/5694 for ODSP capabilities and 
https://github.com/microsoft/FluidFramework/issues/4793 for actual implementation in DeltaManager / hook up in ODSP driver.

I can remove it from this PR, but I plan to add it in next couple days.